### PR TITLE
feat: creates remove admin action for mailer

### PR DESCRIPTION
**Pull request by carloslazo**
In your PR did you:

PR refers to issue #3311 

Let me know if you want me to edit the subject.

`subject: "#{@removed_admin.name} is no longer an admin.")`

or the view message

`<p><%= link_to "#{@removed_admin.name}", url_for(@removed_admin)%> has been removed as a admin for EBWiki.</p>`

  - [x ] Include a description of the changes?
  - [x ] Mention the issue the PR addresses?
  - [ ] Include screenshots of any changes to the UI?
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [x] Add and/or update specs for your code?

### CODE REVIEW

**Comment by carloslazo** [comment]
Also does EBWiki have a slack? I would like to be in the loop.

File: app/services/send_admin_email.rb [hunk 1]
```diff
@@ -5,14 +5,30 @@ class SendAdminEmail
   include Service
 
   def call(user:)
+    # returns an array [false, true] or [true, false]
     previous_admin_status = user.previous_changes['admin']
-    send_new_admin_email(user: user) if previous_admin_status[1]
+    send_new_admin_email(user: user)
```
Comment (rlgreen91) [inline app/services/send_admin_email.rb]: I think you might have left this one in by mistake

File: app/services/send_admin_email.rb [hunk 2]
```diff
@@ -5,14 +5,30 @@ class SendAdminEmail
...
+
+    if previous_admin_status[1]
+      send_new_admin_email(user: user)
+    elsif previous_admin_status[1] == false
```
Comment (rlgreen91) [inline app/services/send_admin_email.rb]: Let's change that to `!previous_admin_status[1]` - I'm surprised rubocop didn't yell about that

**Comment by rlgreen91** [comment]
> Also does EBWiki have a slack? I would like to be in the loop.

Yes, we do, here's the link: https://join.slack.com/t/ebwiki/shared_invite/enQtNzkzMzkwNjY5MTY4LWE5NzQyMWZjZDJlZmMzMzJmYmJjYzVhOTJmZjNkYTYzMzVkOTkwOWUxYTZkY2U4ZGI2MzdkMWM4OTY5NzliNjg

File: app/services/send_admin_email.rb (hunk 1 above)
Comment (carloslazo) [reply app/services/send_admin_email.rb]: Ah, good catch. Thanks.

File: app/services/send_admin_email.rb (hunk 2 above)
Comment (carloslazo) [reply app/services/send_admin_email.rb]: That's cleaner. Thanks.

**Pull request merged**